src/frontends/tensorflow/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenVINO TensorFlow Frontend
+
+The TensorFlow Frontend (TF FE) is a C++ based OpenVINO Frontend component that is responsible for reading and converting a TensorFlow model to an `ov::Model` object
+that further can be serialized into the Intermediate Representation (IR) format.
+
+```mermaid
+flowchart BT
+    model[(model1.pb)]
+    style model fill:#427cb0
+    model2[(model2.pb)]
+    style model2 fill:#427cb0
+    ovtf(OpenVINO Integration with TensorFlow)
+    style ovtf fill:#ffffc2
+    tf_fe(TensorFlow Frontend)
+    style tf_fe fill:#ee9a4d
+    fem(Frontend Manager)
+    mo(Model Optimizer)
+    model --> ovtf --> tf_fe
+    model2 --> mo --> fem --> tf_fe
+    click ovtf "https://github.com/openvinotoolkit/openvino_tensorflow"
+```
+
+Currently, it is only used by [OpenVINO Integration with TensorFlow Frontend](https://github.com/openvinotoolkit/openvino_tensorflow).
+Model Optimizer for now relies on the legacy TensorFlow Frontend developed in Python.
+
+TensorFlow Frontend uses [the common coding style rules](../../../docs/dev/coding_style.md).
```

Comment:
Doesn't seem to be worth mentioning, because there is nothing special with TF FE or any other code inside OpenVINO that follows these rules.

Reply:
Agree that this is not important or looks like much off-top:)